staff/vic/jasmine-users-api.v2.0.0/src/jasmineCLI.spec.js

```diff
@@ -1,4 +1,13 @@
+require('dotenv').config()
+
+const express = require('express')
+
+const userRouter = require('./api/userRouter')
+
+const app = express()
+
```

Comment:
MOVE all express stuff inside the callback of jasmine, so you don't create any early objects until you have passed the tests.